pdns/recursordist/Makefile.am

```diff
@@ -216,3 +222,14 @@ $(MANPAGES):
 	exit 1
 endif
 endif
+
+if !HAVE_LUA_HPP
+# no lua.hpp on solaris :(
+lua.hpp:
```

Comment:
Ok, then how about shipping compat-lua.hpp that we can symlink in to the right projects, to avoid the duplication?